.github/workflows/build.yml

```diff
@@ -3,7 +3,7 @@
 
 name: Python application
 
-on: [push]
+on: [push, pull_request]
```

Comment:
Now the Action is attached to the PR as well, so you can see if a PR builds correctly from the PR itself.

![image](https://user-images.githubusercontent.com/6214761/85199704-8f541d80-b2e9-11ea-9c14-989d7a306894.png)